lib/utils.php

```diff
@@ -0,0 +1,22 @@
+
+<?php
```

Comment:
This has been ported from https://github.com/WordPress/gutenberg/pull/22744 Will rebase this PR when that lands.